frontend/src/store/modules/gems.ts

```diff
@@ -56,12 +60,16 @@ const protoGemToGemMapper = (protoGem: ProtoGem): Gem => {
 const gemsModule: Module<GemsState, RootState> = {
   state: {
     gemsPendingCollection: [],
+    droppedGemId: -1,
```

Comment:
What is this for? Also, can we avoid magic values?

Reply:
changed -1 to undefined
it's not for much, seems like can't define action without mutating state
https://stackoverflow.com/questions/52630866/vuex-actions-that-do-not-need-to-commit-a-mutation